src/main/java/com/xl88/cf401d4/lab/Songr/SongrApplication.java

```diff
@@ -6,7 +6,8 @@
 @SpringBootApplication
 public class SongrApplication {
 
-	public static void main(String[] args) {
+	public static void main(String[] args)
+	{
```

Comment:
`public static void main(String[] args) {`

Reply:
oh...wow...i don't know why it happened...

Reply:
oh i remembered it i just tried to move the "{" down one line, easier for me to look at ...